app/javascript/components/TimeslotRow.jsx

```diff
@@ -4,6 +4,16 @@ import { formatTimeslot } from '../utils/dateFormat'
 import Preference from './Preference'
 
 const TimeSlotRow = ({ timeslot, duration, confirmModal, updatePreference }) => {
+  let rankColor = 'gold'
```

Comment:
I find the switch statement more readable when changing behavior based on a single variable

```javascript
  let rankColor
  let tooltipText

  switch (timeslot.rank) {
    case 1:
      rankColor = 'gold'
      tooltipText = <div>Best<br/>Timeslot</div>
      break
    case 2:
      rankColor = 'silver'
      tooltipText = '2nd Best Timeslot'
      break
    case 3:
      rankColor = 'bronze'
      tooltipText = '3rd Best Timeslot'
      break
  }
```

Reply:
Good call, fixed